tests/integration_tests/with_db/test_buckets.py

```diff
@@ -0,0 +1,10 @@
+from exasol_bucketfs_utils_python.buckets import list_buckets
+
+
+def test_all_default_buckets_are_listed(bucketfs_test_config):
+    assert set(list(list_buckets(
+        base_url=bucketfs_test_config.url,
+        username=bucketfs_test_config.username,
+        password=bucketfs_test_config.password,
+        port=bucketfs_test_config.port
+    ))) == {'default', 'myudfs', 'jdbc_adapter'}
```

Comment:
Maybe only check that default ja contained, not sure of we keep the other forever

Reply:
@tkilias I would adjust the test when this change is introduce in the ITDE, otherwise it may be confusing.
Question: Is there an "easy" way create & remove buckets for python based integration tests?

Reply:
Regarding [this comment](https://github.com/exasol/bucketfs-python/pull/34#discussion_r992998668)

This makes sense to me, that's why I thought introducing the configuration switch with a reasonable default for the tests of this project. At a later point in time it would make sense to use the configuration switch in all tests so we
only need to adjust the port in one place if needed (currently there is a lot of hard coded ones).